generic/tests/cfg/lvm.cfg

```diff
@@ -45,8 +45,6 @@
             sub_type = ioquit
             force_create_image_stg1 = no
             force_create_image_stg2 = no
-            backup_image_before_testing = yes
-            restore_image_after_testing = yes
```

Comment:
I am afraid that the system image may be destroyed and affect the other cases since we did not take a normal way to quit the process. 
So I think that we could clone this system image for this case `lvm_ioquit` to avoid the potential issue.
```
    clone_master = yes
    master_images_clone = image1
    force_image_clone = yes
    remove_image_image1 = yes
```

Reply:
updated, thanks